{'runtime': datetime.datetime(2022, 3, 13, 15, 58, 31, 945374), 'exceptionDetails': [{'year': 2019, 'month': 1, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A8655C700>)}, {'year': 2019, 'month': 2, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A8656C440>)}, {'year': 2019, 'month': 3, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A86580E40>)}]}
{'runtime': datetime.datetime(2022, 3, 13, 15, 59, 27, 365849), 'exceptionDetails': [{'year': 2019, 'month': 1, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7BC5C0>)}, {'year': 2019, 'month': 2, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7CED80>)}, {'year': 2019, 'month': 3, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7E0B80>)}]}
{'runtime': datetime.datetime(2022, 3, 13, 16, 22, 3, 469706), 'exceptionDetails': [{'year': 2019, 'month': 1, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x117dd7540>)}, {'year': 2019, 'month': 2, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x117de8280>)}, {'year': 2019, 'month': 3, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x117dfbe40>)}]}
{'runtime': datetime.datetime(2022, 3, 13, 16, 22, 23, 648708), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 24, 32, 940555), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 25, 56, 916564), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 26, 30, 965264), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 31, 34, 875747), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 44, 33, 246062), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 47, 44, 471721), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 16, 49, 31, 972375), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 17, 49, 54, 275976), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 17, 50, 26, 18714), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 17, 52, 17, 96437), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 17, 59, 37, 850350), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 18, 17, 11, 721655), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 18, 32, 18, 54652), 'exceptionDetails': 'No failed calls in this run.'}
{'runtime': datetime.datetime(2022, 3, 13, 18, 55, 53, 306783), 'exceptionDetails': 'No failed calls in this run.'}
